README.md

```diff
@@ -90,7 +90,7 @@ Support Only For :
 ---
 
 Unfortunately this library has some limitation
-* It does not support private fields. Make sure your structs fields you intend to generate fake data for are public, it would otherwise trigger a panic. You can however omit fields using a tag skip `faker:"-"` on your private fields.
+* It does not support private fields. Make sure your structs fields you intend to generate fake data for are public, it would otherwise trigger a panic. You can however omit fields using a tag skip `faker:"-"` on your public fields.
```

Comment:
I think both for private and public should be added the ignore tag `faker:"-"`. I recommend removing this instead.